addon/belongs-to-sticky.js

```diff
@@ -17,13 +16,13 @@ var recordHasId = function (record) {
  *
  * @returns {Ember.computed} relationship
  */
-var belongsToSticky = function () {
-  var computed = DS.belongsTo(...arguments);
-  var meta = computed.meta();
+const belongsToSticky = function () {
+  let computed = DS.belongsTo(...arguments);
+  let meta = computed.meta();
   meta.sticky = true;
-  return Ember.computed({
+  return computed({
```

Comment:
This doesn't seem right, it's supposed to use Ember.computed, but is using the computed variable fromthe DS.belongsTo call a few lines above.

Reply:
Ah, i suppose that might be te problem I ran into, will try to fix it.

Reply:
So, I have fixed the problem, and prefixed the computed from DS.belongsTo with an underscore, and it is all working now.